chunky/src/java/se/llbit/chunky/resources/OctreeFileFormat.java

```diff
@@ -32,8 +32,9 @@
    * Load octrees and grass/foliage textures from a file.
    *
    * @param in input stream for the file to load the scene from.
+   * @param forceNodeBased Forces the octrees to be loaded as NodeBasedOctree
    */
-  public static OctreeData load(DataInputStream in) throws IOException {
+  public static OctreeData load(DataInputStream in, boolean forceNodeBased) throws IOException {
     int version = in.readInt();
     if (version != OCTREE_VERSION) {
```

Comment:
Did the file format stay the same? If not, the `OCTREE_VERSION` needs to be increased.

Reply:
The format stayed the same, no need to update the OCTREE_VERSION